R/trex.R

```diff
@@ -28,15 +28,15 @@
 #'   long-form description of the extension. HTML text can be used for the
 #'   latter.
 #' @param author_name,author_email,author_organization,website Details about the
-#'   extension author and project website.
+#'   extension author and the project website.
 #' @param source_location The source location, which is the URL of the server
 #'   that hosts the web page that in turns interacts with Tableau. If `NULL`
 #'   (the default, and a good option) then **shinytableau** will autodetect
 #'   the source location.
 #' @param icon_file The path to the icon file to be used for the extension. The
 #'   image used as an icon file must be a 70x70px PNG file.
 #' @param permissions A setting that determines the level of permissions. The
-#'   default `"full_data"` is the default and currently the only available
+#'   value `"full_data"` is the default and is currently the only available
```

Comment:
We should also allow permissions to be NULL (though `full_data` should remain the default). I have a `permissions <- match.arg(permissions)` in there, but that's wrong; if anything, it should be checking that `permissions` is either `NULL` or `"full data"`.